live/global.ini

```
@@ -65806,12 +65806,12 @@ spacecargo_illegal_desc_0006=Hey! Wer hat Lust auf Wrackerkundung? Da gibt es ei
 spacecargo_illegal_from=~mission(Contractor)
 spacecargo_illegal_location_0001=die Überreste
 spacecargo_illegal_location_0002=der letzte Kontaktpunkt
-spacecargo_illegal_location_0003=die letzte bekannte Position der ~mission(Ship).
-spacecargo_illegal_location_0004=das Wrack der ~mission(Ship).
+spacecargo_illegal_location_0003=die letzte bekannte Position der ~mission(Ship)
+spacecargo_illegal_location_0004=das Wrack der ~mission(Ship)
 spacecargo_illegal_location_0005=die Tötungsstelle
-spacecargo_illegal_obj_long_01=Begib dich zu ~mission(Location), um das Paket abzurufen.
-spacecargo_illegal_obj_long_02=Erhalte die Lieferung von ~mission(Item) aus dem Wrack.
-spacecargo_illegal_obj_long_03=Liefere es an ~mission(Destination).
+spacecargo_illegal_obj_long_01=Begib dich zu ~mission(Location), um das Paket abzurufen
+spacecargo_illegal_obj_long_02=Erhalte die Lieferung von ~mission(Item) aus dem Wrack
```

Da bin ich nicht ganz sicher, finde aber es hört sich besser an...
```suggestion
spacecargo_illegal_obj_long_02=Hole die Lieferung von ~mission(Item) aus dem Wrack
```